tap_kit/client.py

```diff
@@ -30,7 +51,7 @@ def requests_method(method, request_config, body):
             json=body)
 
     @backoff.on_exception(backoff.expo,
-                          RateLimitException,
+                          CONN_RETRY_ERRORS + RetryableException,
```

Comment:
Just double-checking that this is valid syntax :) Does it need to be a tuple/list?